nowcasting_dataset/data_sources/fake/batch.py

```diff
@@ -3,7 +3,9 @@
 Wanted to keep this out of the testing frame works, as other repos, might want to use this
 """
 
-from typing import Optional
+import datetime
```

Comment:
i dont like the 'datetime.datetime' below, I think its a bit unclear (maybe its just what im no use to)

and prefer just importing the modules we need, rather all of it. 

Could you do
`from datetime import datetime`

Reply:
Good suggestion! Done!